packages/web/src/common/store/player/sagas.ts

```diff
@@ -155,13 +160,27 @@ export function* watchPlay() {
         ? apiClient.makeUrl(`/tracks/${encodedTrackId}/stream`, queryParams)
         : null
 
+      const isLongFormContent =
+        track.genre === Genre.PODCASTS || track.genre === Genre.AUDIOBOOKS
+
       const endChannel = eventChannel((emitter) => {
         audioPlayer.load(
           track.track_segments,
           () => {
             if (onEnd) {
               emitter(onEnd({}))
             }
+            if (isNewPodcastControlsEnabled && isLongFormContent) {
```

Comment:
does this also get called in mobile? just wondering if we get a double call to setTrackPosition "completed"?